dev-tool/src/list_net/evaluate.rs

```diff
@@ -0,0 +1,95 @@
+#![cfg(not(tarpaulin))]
+
+use std::{iter, ops::Add, path::PathBuf};
+
+use anyhow::{bail, Context, Error};
+use indicatif::{ProgressBar, ProgressStyle};
+use rayon::iter::{ParallelBridge, ParallelIterator};
+use structopt::StructOpt;
+
+use xayn_ai::list_net::{ndutils::kl_divergence, DataSource as _, ListNet};
+
+use super::data_source::{DataSource, InMemorySamples};
+use crate::{exit_code::NO_ERROR, utils::progress_spin_until_done};
+
+/// Runs a single evaluation pass on a ListNet.
+#[derive(StructOpt, Debug)]
+pub struct EvaluateCmd {
+    /// A file containing samples we can train and evaluate on.
+    #[structopt(long)]
+    samples: PathBuf,
+
+    /// File containing a ListNet parameter set.
+    ///
+    /// E.g. `list_net.binparams`
+    #[structopt(long)]
+    parameters: PathBuf,
+
+    /// The percent of samples to use for evaluation.
```

Comment:
```suggestion
    /// The percentage of samples to use for evaluation.
```
but maybe `fraction` would be a better word, since your default value is `0.2` rather than `20` percent